### ISSUE TITLE
Can we have chapter links in adoc?

### ISSUE THREAD

**Issue by unfa** [zynaddsubfx/user-manual#3]
I am wondering how can or should we handle self-linking.

Say we briefly mention Macro Learn in a chapter about user interface - it'd be good to add a reference linking to the chapter devoted to Macro Learn if the user wants to go there.

I think that ideal references would work in two ways:

Provide textual information that the user can process and execute himself ("More in chapter 8.3") and provide a hypertext link that will let him jump there immediately or open the relevant chapter in a new browser tab.

I don't know how can we do this in Asciidoc. Any ideas?

**Comment by fundamental** [comment]
I think you're looking for http://asciidoctor.org/docs/user-manual/#internal-cross-references . There might be some more work to expand upon this though.

**Comment by unfa** [comment]
Yeah, I tried using that, but it doesn't seem to do what I wanted.
I would guess that from the level of `Manual.adoc` all references placed in chapter files should be interpreted like they are in a single document. I tried this:

     TIP: you can learn more about this in the <<Global Settings>> chapter.

trying to reference a chapter name in `Settings.adoc`:

     === Global Settings
     image::imgs/Selection_422.png[]

But it links me back to the top of the document instead. I probably didn't do it right.

**Comment by fundamental** [comment]
> Yeah, I tried using that, but it doesn't seem to do what I wanted.

That's odd.
I guess I'll have to experiement around with that feature a bit since I
haven't use those sorts of internal-links within asciidoc before.

**Comment by celestehorgan** [comment]
https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links

By default, most "cross references" are actually cross references _within the same .adoc file_. To cross reference to another adoc file's section, you need to be using Asciidoctor.

To make this work, we'd need to create an anchor link to the section in Settings.adoc :)
